js_utils/text_validation.js

```diff
@@ -93,8 +93,8 @@ export class TextValidation {
         if (error == false) {
           const year = parseInt(year_code);
           const day = parseInt(day_code);
-          if (year == 20) {
-            // Year is always 20
+          if (year > -1 && year < 100) {
+            // Year is 00 to 99
             if (day > 0 && day < 367) {
               // Day is between 1 to 366
               error = false;
```

Comment:
it looks like the build is failing due to loss of coverage on line 105.  In general, I would recommend refactoring this section similar to my suggestion in the last Pull Request for your similarly structured logic around determining whether the `Save` button should be enabled in the Pop Up dialogs